src/README.md

```diff
@@ -33,25 +37,31 @@ If you only want to use JavaMOP out of the box please refer to ../INSTALL.md
 
  ```git clone https://github.com/runtimeverification/javamop.git```
 
- You can also download a "zip" or "tar.gz" from [here](https://github.com/runtimeverification/javamop/releases) and unzip to your favorite location.
+ You can also download a "zip" or "tar.gz" from
+ [here](https://github.com/runtimeverification/javamop/releases) and
+ unzip to your favorite location.
```

Comment:
Hi,

I&#39;d like to add you to my professional network on LinkedIn.

- Qingzhou

Accept: https://www.linkedin.com/blink?simpleRedirect=0QdjANc3kPej0Td38MejsRdPARfkh9rCZFt65QqnpKqioRrQwPoAJdizRVpkJApn9xq7cCt7dBtmtvpnhFtCVFfmJB9C5QoORBt6BSrCAJt7dBtmsJr6RBfmtKqmJzon9Q9ClQqnpKimtBkClOs3Rx9DkObjdMpP9IqjpFbmZWqjxFrPRBfP9SbSkLrmZzbCVFp6lHrCBIbDtTtOYLeDdMt7hE&msgID=I5975790272099946496_500&markAsRead=


You received an invitation to connect. LinkedIn will use your email address to make suggestions to our members in features like People You May Know. Unsubscribe here: https://www.linkedin.com/blink?simpleRedirect=tj8JcT1DcCNFdCAJrTFFe6BLfmhFpipdrjhUoRB3hRcTgCNxrAwTp4BikQZCh7pKmmhIs79LnQJGpAllk4V6djwQe4lWc4FMd35nd5t3pCdInQt6d4AQoAtKlT5ciRZetnxepThmclxUmmN9oRZ3hkxcm65JpShmuA5flA5Vilp6jl1MlD57j6RncS9Pcn8Vqz1JlBthpAAJikFDhPpFgBpbiSdJhzlRlk51gntxqChWh4ZytSZShl51fmhFrSMCt7dBtmtvpnhFtCVFfmJB9B4ScSh5hSxmhD9JhB51fmVBqSZkp6BJ9CVRr3RQ9DkObjdMpP9IqjpFbmZWqjxFrPRBfP9SbSkLrmZzbCVFp6lHrCBIbDtTtOYLeDdMt7hE&amp;msgID=I5975790272099946496_500&amp;markAsRead= Learn why we included this at the following link: https://www.linkedin.com/blink?simpleRedirect=e3wTd3RAimlIoSBQsC4Ct7dBtmtvpnhFtCVFfmJB9CNOlmlzqnpOpldOpmRLt7dRoPRx9DkObjdMpP9IqjpFbmZWqjxFrPRBfP9SbSkLrmZzbCVFp6lHrCBIbDtTtOYLeDdMt7hE&msgID=I5975790272099946496_500&markAsRead=
&copy; 2014, LinkedIn Corporation. 2029 Stierlin Ct. Mountain View, CA 94043, USA